packages/react-i18n/src/index.tsx

```diff
@@ -69,8 +69,14 @@ export const withI18n = createHigherOrderComponent< I18nReact >( ( InnerComponen
  * @returns The context value with bound translation functions
  */
 function makeContextValue( localeData?: LocaleData ): I18nReact {
-	const i18n = createI18n( localeData );
 	const i18nLocale = localeData?.[ '' ]?.localeSlug ?? 'en';
+
+	if ( ! localeData ) {
+		return { __, _n, _nx, _x, isRTL, i18nLocale };
```

Comment:
As discussed on a P2, the `i18nLocale`'s default value `en` will be wrong if the upstream global `i18n` instance is actually localized.

A good alternative would be to not pass the `i18nLocale` prop in this case at all, or completely remove it from the context value.

Reply:
I'm going to remove `i18nLocale` since currently there's currently nothing depending on it. Not ruling out that it won't eventually make it's way back in some form though.